### PR TITLE
Add alert component

### PR DESCRIPTION
**Pull request by ravikumar1002**
This component has different kind of alerts like success,  info, warning, error,  alert with button, icon-alert.

### CODE REVIEW

**Comment by Rohanmond** [comment]
Looks Good! ready for merge

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+.alert-container {
+  max-width: 50rem;
+  display: inline-block;
```
Comment (anandsr21harshit) [inline components/alert/alert.css:3]: using flex would be a better option.

**Review by anandsr21harshit — APPROVED** [review]
Add a margin-bottom, UI will look good.

File: components/alert/alert.css (hunk 1 above)
Comment (ravikumar1002) [reply components/alert/alert.css:3]: yeah I could use but it , but I tried to make separate class, and flex would change my different element also,  that's why 
 I used this.
Thanks for reviewing my PR

**Review by Rohanmond — APPROVED** [review]

**Pull request merged**